lochness/email/__init__.py

```diff
@@ -19,10 +29,146 @@ def send(recipients, sender, subject, message):
     s.sendmail(sender, recipients, msg.as_string())
     s.quit()
 
+
```

Comment:
Did you follow a template for writing `send_detail_google()`? If yes, can you refer me to that?

Reply:
I've followed the `send` function in the `lochness.email`

https://www.bootdey.com/snippets/view/bs4-Email-alert

Reply:
From your PR description:

> The original version of Lochness required mailx to be configured by the system admin, and this is not always the case for U24 production servers. 

So does `smtplib.SMTP()` exploit `mailx` which is not working at one of the U24 servers? I am trying to understand why we set up Google email.